src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2147,7 +2151,7 @@
         "graphTraversals": {
           "description": "An array of one or more unique 'graphTraversal' objects.",
           "type": "array",
-          "minItems": 0,
+          "minItems": 1,
```

Comment:
>minItems [](start = 11, length = 8)

same not, s/be 0

Reply:
Yes 0 is right. Also note that the default for both of these is `[]` -- which is right -- and the default shouldn't violate the `minItems` constraint.

---
In reply to: [276013562](https://github.com/Microsoft/sarif-sdk/pull/1428#discussion_r276013562) [](ancestors = 276013562)

Reply:
Thanks, i will revert this and mention this as spec issue in excel sheet.

---
In reply to: [276015944](https://github.com/Microsoft/sarif-sdk/pull/1428#discussion_r276015944) [](ancestors = 276015944,276013562)